client/my-sites/email/email-management/email-home.jsx

```diff
@@ -117,14 +118,18 @@ class EmailManagementHome extends React.Component {
 			page( emailManagementTitanSetUpMailbox( selectedSite.slug, selectedSite.domain ) );
 		}
 
+		const emailListActive = isUnderEmailInbox( currentRoute ) ? null : (
+			<EmailListActive
+				domains={ domainsWithEmail }
+				selectedSiteSlug={ selectedSite.slug }
+				currentRoute={ currentRoute }
+				selectedSiteId={ selectedSiteId }
+			/>
+		);
+
 		return this.renderContentWithHeader(
 			<>
-				<EmailListActive
-					domains={ domainsWithEmail }
-					selectedSiteSlug={ selectedSite.slug }
-					currentRoute={ currentRoute }
-					selectedSiteId={ selectedSiteId }
-				/>
+				{ emailListActive }
```

Comment:
Not mandatory but we could keep that component inlined like this:

```suggestion
				{ isUnderEmailInbox( currentRoute ) && <EmailListActive ... }
```

Reply:
Yes, I like it more than you said, we can see at a glance what is everything.